pom.xml

```diff
@@ -1501,4 +1501,4 @@
             </build>
         </profile>
     </profiles>
-</project>
+</project>
```

Comment:
this can be reverted

Reply:
Yes and no. I can revert it, which would increase the build since is wrongly filled. The cache is selected by the hash of all pom's.